documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -4,33 +4,33 @@ This document outlines how to use the Performance Toolkit SDK (SDK) to create
-    - [Configuring Your Project](#configuring-your-project)
-        * [Adding the Microsoft.Performance.SDK NuGet Package](#adding-the-microsoftperformancesdk-nuget-package)
-        * [Picking your SDK version](#picking-your-sdk-version)
-        * [Installing WPA for Debugging](#installing-wpa-for-debugging)
-        * [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
-2. [Creating a ProcessingSource](#creating-a-processingsource)
-    - [Creating a ProcessingSource class](#creating-a-processingsource-class)
-    - [Decorating your ProcessingSource with the ProcessingSourceAttribute](#decorating-your-processingsource-with-the-processingsourceattribute)
-    - [Decorating your ProcessingSource with a DataSourceAttribute](#decorating-your-processingsource-with-a-datasourceattribute)
-    - [Implementing the required ProcessingSource methods](#implementing-the-required-processingsource-methods)
-    - [(Optional) Adding About Information](#adding-about-information)
-3. [Choosing a Plugin Framework](#choosing-a-plugin-framework)
-4. [Creating a CustomDataProcessor and Tables](#creating-a-customdataprocessor-and-tables)
+* [Creating the Project](#creating-the-project)
+  * [Requirements](#requirements)
+  * [Creating Your Project](#creating-your-project)
+  * [Configuring Your Project](#configuring-your-project)
+    * [Adding the Microsoft.Performance.SDK NuGet Package](#adding-the-microsoftperformancesdk-nuget-package)
+    * [Picking your SDK version](#picking-your-sdk-version)
+    * [Installing WPA for Debugging](#installing-wpa-for-debugging)
+    * [Setup for Debugging Using WPA](#setup-for-debugging-using-wpa)
+* [Creating a ProcessingSource](#creating-a-processingsource)
+  * [Creating a ProcessingSource class](#creating-a-processingsource-class)
+  * [Decorating your ProcessingSource with the ProcessingSourceAttribute](#decorating-your-processingsource-with-the-processingsourceattribute)
+  * [Decorating your ProcessingSource with a DataSourceAttribute](#decorating-your-processingsource-with-a-datasourceattribute)
+  * [Implementing the required ProcessingSource methods](#implementing-the-required-processingsource-methods)
+  * [Adding About Information](#adding-about-information)
+* [Choosing a Plugin Framework](#choosing-a-plugin-framework)
+* [Creating a CustomDataProcessor and Tables](#creating-a-customdataprocessor-and-tables)
 
 ---
 
 ## Creating the Project
 
 Plugins are created as C# class libraries that get dynamically loaded by the SDK runtime. To begin your plugin creation, we will first walk through creating a C# project.
 
-> :information_source: The SDK team is actively working on creating a dotnet template to simplify creating your project and writing the neccessary plugin boilerplate code. 
+> ⚠️ The SDK team will be creating [(see issue)](https://github.com/microsoft/microsoft-performance-toolkit-sdk/issues/190) a dotnet template to simplify creating your project and writing boilerplate code. 
```

Comment:
Added a comment to the issue in order to make sure we alter the documentation as needed.